modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,39 @@
+[id="overview-of-openshift-gitops"]
+= Overview of OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: overview-of-openshift-gitops
+toc::[]
+
+:FeatureName: OpenShift GitOps
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Use Red Hat OpenShift GitOps to create repeatable processes for managing OpenShift Container Platform clusters and applications across multi-cluster Kubernetes environments. When you lose time in deployment and release cycles, Red Hat Openshift GitOps is a tool that handles and automates the fast pace and complex deployments. The Red Hat Openshift GitOps’ workflow pushes from application tp development environment to testing environment to staging to production.
+
+Red Hat Openshift GitOps is a declarative way to implement continuous deployment for cloud knative applications. Red Hat Openshift GitOps has a Git repository that contains a declarative description of the infrastructure you need in your specified environment. This Git repository also contains an automated process to make your environment match the described state. Basically,  Red Hat Openshift GitOps either deploys a new application or updates an existing one, so you only need to update the repository.  Red Hat Openshift GitOps handles and automates everything else!
```

Comment:
This entire paragraph complements other sections that are explained in a much better manner in the section below.
I have made suggestions on where they can add more value.
Remove the word 'Basically' and the exclamation mark at the end.